srcpkgs/swaync/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'swaync'
+pkgname=swaync
```

Comment:
follow the upstream naming (SwayNotificationCenter)

Reply:
followed this actually. https://aur.archlinux.org/packages/swaync  
author used swaync too

Reply:
we aren't the AUR. see [here](https://github.com/void-linux/void-packages/blob/master/Manual.md#programs)

Reply:
author used swanync only in readme file.  
https://github.com/ErikReider/SwayNotificationCenter#sway-usage

Reply:
Updated repo name. thanks.